### PR TITLE
Animation works

### PR DESCRIPTION
**Pull request by jvn1228**
So merge it
Now we create Toystory 4  with this?

### CODE REVIEW

File: src/hallway/core.cljs [hunk 1]
```diff
@@ -15,3 +15,15 @@
     (set! (.-fillStyle ctx) color)
     (.fillRect ctx box-x box-y 100 100)))
 
+(defn erase-ctx
+	[]
+	(let [canvas (.getElementById js/document "canvas")
+        ctx (.getContext canvas "2d")]
+    (.clearRect ctx 0 0 (.-innerWidth js/window) (.-innerHeight js/window))))
```
Comment (chr1sto14) [inline src/hallway/core.cljs]: I think this line needs to be tabbed over.

File: src/hallway/core.cljs [hunk 2]
```diff
@@ -15,3 +15,15 @@
...
+
+(defn animate-box
+	[init-x init-y timeout]
+	(when (< init-x (-(.-innerWidth js/window) 100))
```
Comment (chr1sto14) [inline src/hallway/core.cljs]: The minus sign needs a space after it.

**Comment by chr1sto14** [comment]
bump

**Comment by jvn1228** [comment]
Uh changed

**Pull request merged**